mlops_ml_models/models_template_notebook.ipynb

```diff
@@ -2,9 +2,184 @@
  "cells": [
```

Comment:
Are any of those changes actually needed or it's just output from the notebook run?

Reply:
Ah, this is just output from the notebook run, this has to do with the first cell.

Reply:
The outputs will change each time we run the notebook and GitHub will pick it up. It has always been that way. I have not changed much, I only changed the arrangement of the notebook cells to look the way we had it in our first demo cos I did not download the notebook we used in the first demo, so this notebook looks better. And I added the picture of the Architectural diagram to the first cell. So no major changes. 

If I should rerun the notebook and push now, GitHub will assume I changed the cells which is not the case.

Reply:
Sorry, I misread your message, I will try to find a way to clear it.